pkg/crc/api/client/types.go

```diff
@@ -20,15 +20,17 @@ type StartResult struct {
 }
 
 type ClusterStatusResult struct {
-	CrcStatus        string
-	OpenshiftStatus  string
-	OpenshiftVersion string `json:"OpenshiftVersion,omitempty"`
-	PodmanVersion    string `json:"PodmanVersion,omitempty"`
-	DiskUse          int64
-	DiskSize         int64
-	RAMUse           int64
-	RAMSize          int64
-	Preset           preset.Preset
+	CrcStatus         string
+	OpenshiftStatus   string
+	OpenshiftVersion  string `json:"OpenshiftVersion,omitempty"`
+	MicroShiftStatus  string
+	MicroShiftVersion string `json:"MicroShiftVersion,omitempty"`
+	PodmanVersion     string `json:"PodmanVersion,omitempty"`
```

Comment:
This becomes as mess on the receiving end.

Just a `Version` and `Status` will do, as the `Preset` is the key value to determine the meaning